docs/content/tutorials/tutorial-kerberos-hadoop.md

```diff
@@ -0,0 +1,125 @@
+<!--
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+-->
+# Tutorial: Configuring Druid to use a Kerberized Hadoop as Deep Storage
+
+
+## Hadoop Setup
+
+Following are the configurations files required to be copied over to Druid conf folders:
```

Comment:
You may want to say "files required to be copied over to conf directory in the Druid package root" to clarify where the conf directory is located